### DISCUSSION TITLE
Allow to Disable/Enable Motion Activated Lights on Floodlight Cam Plus

### DISCUSSION THREAD

**Discussion by iceman73** [tsightler/ring-mqtt#705] [Feature Requests]
In the Ring app, you can Enable and Disable the Motion-Activated Lights on the Ring Cam Flood light. I would like the ability to enable and disable the Motion-Activated Lights so when we are outside watching TV, HA can disable this feature rather than me whipping out the phone and going into the RING app to do this. In Fact this function is the last thing I need here to not have to use the ring app anymore.

**Comment by tsightler** [comment]
This is a duplicate of https://github.com/tsightler/ring-mqtt/discussions/320.  It's closer than it's ever been as I was able to work out the API as noted in the latest update, but, unfortunately, there are more pressing issues at the moment.
